packages/editor-ui/src/components/CodeNodeEditor/linter.ts

```diff
@@ -559,6 +559,75 @@ export const useLinter = (
 			});
 		});
 
+		/**
```

Comment:
Do we have tests for this file?

Reply:
There's no tests for this file. Seeing as there's no setup already I decided to skip so I can move to other higher prio stories, else let me know if worth it and I'll see how to add them.